### DISCUSSION TITLE
How to create a firework explosion effect

### DISCUSSION THREAD

**Discussion by mudkipdev** [Minestom/Minestom#2328] [General]
I want colored particles from the firework with the sound

**Comment by cosrnic** [comment]
For the firework create an item with fireworks item component then create an entity with the firework type and set the firework info in its meta. To explode the firework call `triggerStatus` with `0x17` (https://wiki.vg/Entity_statuses#Firework_Rocket), all the sounds are something you'll have to do yourself iirc
